docs/more-info/webhooks.mdx

```diff
@@ -1,8 +1,157 @@
 ---
 title: "Webhooks"
-description:
-  "We're hard at work building out our Webhooks so you can subscribe to your
-  users' health data."
+description: "Receive user information as soon as it becomes available"
 ---
 
-<Warning>**Check back soon!**</Warning>
+## Overview
+
+Usually, when an application wants to consume information from another it makes a request to
+obtain such information.
+
+Webhooks turn this around: the producer of information calls the consumer when data is available.
+This way, the consumer doesn't have to be poll the producer to obtain the information.
+
+So, to enable this integration approach with Metriport:
+
+1. Expose a public endpoint on your app so that Metriport can call it when user data
+   is available (see format below).
+1. Set this endpoint URL on the [settings page](https://dash.metriport.com/settings)
+1. This will generate a webhook key, that should be used to authenticate Metriport's requests on
+   your app's endpoint (webhook)
+
+## The Webhook Format
+
+The webhook format specification is as follows:
+
+1. Public endpoint accessible from the internet;
+1. Accepts a `POST` HTTP request;
+1. Verifies requests by comparing the HTTP header `x-webhook-key` with the webhook key available
+   on the [settings page](https://dash.metriport.com/settings);
+1. Accepts and responds to a [`ping` message](#the-ping-message);
+1. Accepts and processes [user data messages](#user-data-message).
```

Comment:
Consistency comments:

- `1. 2. 3.`

Wording comments:

- "... on the settings page in the developer dashboard."

Reply:
> 1. 2. 3.
Markdown sets the numbering automatically, we don't need to maintain the item's numbers in order

Reply:
Didn't know this!